po/pt.po

```diff
@@ -0,0 +1,914 @@
+msgid "Commands"
+msgstr "Comandos"
+
+#. In this context meaning "work environment".
+#: src/text.vala:465
+msgid "Desktop"
+msgstr "Ambiente de Trabalho"
+
+#: src/text.vala:467
+msgid "Behavior"
+msgstr "Comportamento"
+
+#: src/text.vala:468
+msgid "Appearance"
+msgstr "Aparência"
+
+#: src/text.vala:469
+msgid "Notifications"
+msgstr "Notificações"
+
+#: src/text.vala:471
+msgid "_Protocol:"
+msgstr "_Protocolo:"
+
+#: src/text.vala:472
+msgid "Both IPv4 and IPv6"
+msgstr "Ambos IPv4 e IPv6"
+
+#: src/text.vala:473
+msgid "IPv4 only"
+msgstr "Somente IPv4"
+
+#: src/text.vala:474
+msgid "IPv6 only"
+msgstr "Somente IPv6"
+
+#: src/text.vala:476
+msgid "Show _Offline Members"
+msgstr "Mostrar _Membros Desligados"
+
+#: src/text.vala:477
+msgid "Sort by _Name"
+msgstr "Ordenar por _Nome"
+
+#: src/text.vala:478
+msgid "Sort by _Status"
+msgstr "Ordenar por _Estado"
+
+#: src/text.vala:480
+msgid "C_onnect automatically on startup"
+msgstr "_Conectar automaticamente ao iniciar"
+
+#: src/text.vala:481
+msgid "_Reconnect automatically when the connection is lost"
+msgstr "_Reconectar automaticamente quando a conexão for perdida"
+
+#: src/text.vala:482
+msgid "_Disconnect on quit"
+msgstr "_Desconectar ao sair"
+
+#: src/text.vala:484
+msgid "Use _dark theme variant"
+msgstr "Usar o tema _escuro"
+
+#: src/text.vala:486
+msgid "Show _indicator"
+msgstr "Mostrar _Indicador"
+
+#: src/text.vala:488
+msgid "Display notification when the connection is l_ost"
+msgstr "Exibir notificação quando a conexão é _perdida"
+
+#: src/text.vala:489
+msgid "Display notification when a member _joins"
+msgstr "Exibir notificação quando um membro se _junta"
+
+#: src/text.vala:490
+msgid "Display notification when a member _leaves"
+msgstr "Exibir notificação quando um membro _sai"
+
+#: src/text.vala:491
+msgid "Display notification when a member comes o_nline"
+msgstr "Exibir notificação quando um membro fica _ligado"
+
+#: src/text.vala:492
+msgid "Display notification when a member goes o_ffline"
+msgstr "Exibir notificação quando um membro fica _desligado"
+
+#: src/text.vala:494
+msgid "Error connecting"
+msgstr "Erro ao conectar"
+
+#: src/text.vala:495
+msgid "Hamachi login failed"
+msgstr "O login no Hamachi falhou"
+
+#: src/text.vala:496
+msgid "No internet connection"
+msgstr "Sem ligação à Internet"
+
+#: src/text.vala:498
+msgid "Add"
+msgstr "Adicionar"
+
+#: src/text.vala:499
+msgid "Add Command"
+msgstr "Adicionar Comando"
+
+#: src/text.vala:500
+msgid "_Add"
+msgstr "_Adicionar"
+
+#: src/text.vala:501
+msgid "Edit"
+msgstr "Editar"
+
+#: src/text.vala:502
+msgid "Edit Command"
+msgstr "Editar Comando"
+
+#: src/text.vala:503
+msgid "_Label:"
+msgstr "_Etiqueta:"
+
+#: src/text.vala:504
+msgid "IPv_4 command:"
+msgstr "Comando IPv_4:"
+
+#: src/text.vala:505
+msgid "IPv_6 command:"
+msgstr "Comando IPv_6:"
+
+#: src/text.vala:506
+msgid "Use %A for address and %N for nickname"
+msgstr "Usar %E para endereço e %A para alcunha"
```

Comment:
Variables (`%A` and `%N`) should not be translated.

Reply:
And done

Thank you :)